### DISCUSSION TITLE
Can't Sort items or change page

### DISCUSSION THREAD

**Discussion by amynewsam** [EsriPS/arcgis-assistant-feedback#67] [Q&A]
I'm trying to copy some web maps and associated features services from one organization to another.  I have found conflicting info on whether to use AGOL Assistant or ArcGIS Assistant, but I tried AGOL Asst with it's option to do a "full copy" which is supposed to "replicate the original service in the destination account and harvest all of its associated data" and while it said everything copied, I was still not able to get the web maps to load the associated layers, so I am trying again with ArcGIS Assistant.  My problem is that I have many files, and for some reason ArcGIS Assistant will not allow me to sort the layers by title (it says it is, but it isn't) or click to the next page of content or change the number of items per page.  Any help appreciated. Thanks.

**Comment by jpeterson** [comment]
Hey @amynewsam, thanks for noting this issue. We are in the process of moving Assistant over to use [Calcite Design System](https://developers.arcgis.com/calcite-design-system/), which will resolve this problem.

In the meantime, I'd recommend using Assistant's search as a temporary workaround – using the search would allow you to narrow results to avoid needing to go to the next page.

**Reply by jpeterson** [reply]
Note that I've also created an issue (#69) from this discussion

**Reply by amynewsam** [reply]
Thanks for those suggestions.  :)

**Comment by jpeterson** [comment]
This was resolved in the September 2023 update: https://guide.assistant.esri-ps.com/docs/changelog#fixed

**Comment by amynewsam** [comment]
[like]  Amelia Newsam reacted to your message:
________________________________
